init.lua

```diff
@@ -367,24 +354,44 @@ function RECV_MSG.ReceivedItems(msg)
 end
 
 
+function RECV_MSG.RoomInfo(msg)
+	local checksum_info = msg["datapackage_checksums"]
+	local game_list = {}
+	for game, checksum in pairs(checksum_info) do
+		if Cache.ChecksumVersions:get(game) == checksum then
+			-- checksum for this game is the same as the cached version
+		else
+			table.insert(game_list, game)
+		end
```

Comment:
```suggestion
		-- checksum for this game is not the same as the cached version
		if Cache.ChecksumVersions:get(game) ~= checksum then
			table.insert(game_list, game)
		end
```

I think we can also skip the game called "Archipelago".

Reply:
Probably best to leave it in, since it might be used for something later on, and also it's tiny.